inst/pages/14_alpha_diversity.qmd

```diff
@@ -105,7 +105,7 @@ library(scater)
 plotColData(tse, 
```

Comment:
Please check that you have modified only mia functions.


plotColData, plotRowData, plotExpression and plotReducedDim comes from the scater package. Do not modify those lines